content/blog/2023-03-14-mining-ethereum-classic-with-an-ipollo-asic-through-2miners-pool/index.md

```diff
@@ -0,0 +1,178 @@
+
+As a consequence, by both hashrate and market cap, ETC is now the largest proof of work smart contracts blockchain in the world.
+
+As many individuals and businesses are planning to enter the mining business, and to mine ETC in particular, we will create an infrastructure series to explain how to start mining Ethereum Classic.
+
+In this post we explain how to mine ETC with an iPollo ASIC mining rig, using the 2miners mining pool.
+
+![ETC hasrate growth.](./intro.png)
+
+## 1. Buy the iPollo Mining Rig
+
+The first step is to get the ASIC mining rig from iPollo directly (they are based in Hong Kong and ship worldwide) or from Amazon if you prefer.
+
+iPollo has three models that work with ETC. The one we are going to present today is the V1 Mini SE Plus, which you may buy on these links:
+
+iPollo (Hong Kong):
+https://ipollo.com/products/ipollo-v1-mini-se-plus
+
+Amazon (USA):
+https://www.amazon.com/dp/B0BHPLSX7V
+
+*Disclaimer: we have no relationship with iPollo or Amazon, nor do we get any commissions or affiliate fees.*
+
+![Buy at iPollo or Amazon.](./1.png)
+
+## 2. iPollo ASIC Rig Unboxing and Necessary Components
+
+When you get the iPollo V1 Mini SE Plus you will get four things:
+
+1. The iPollo V1 Mini SE Plus mining machine
+2. A transformer
+3. An electric AC cable
+4. A card with a QR code to their help page for instructions
+
+The other three components that need to be provided by you to set up the mining operation are:
+
+1. An ethernet cable to connect it to your local LAN
+2. A local LAN, which is your internet connection from home or office (we use Cox in this example)
+3. Your computer so you can manage the iPollo ASIC app from your browser
+
+*Note: Because we use a MacBook Pro computer we needed to connect the iPollo to our local LAN using the Ethernet, but if you use a Windows computer, you may find the iPollo rig over the WiFi networks list in your PC.*
+
+![iPollo unboxing.](./14.png)
+
+## 3. iPollo ASIC Rig Setup and Connection
+
+To set up your mining rig, you need to connect the ethernet cable to the iPollo ASIC miner, and then connect the other end to your LAN modem.
+
+Then, you need to connect the transformer to the iPollo ASIC miner as shown in the image, and then connect the AC cable to the transformer and to the wall electric feed.
+
+![iPollo connection.](./3.png)
+
+## 4. Get the iPollo ASIC Mining Rig URL in Your Local LAN
+
+Now that your mining rig is connected and powered, you have to find its IP address in your local LAN so you may access its app from your browser from your computer (your computer needs to be connected through ethernet cable or WiFi to the local LAN as well).
+
+To do this, go to the IP address of your LAN modem (this may be Googled or requested to your internet provider if you don’t have it) access the app of your local LAN modem, find the iPollo machine in the “connected devices” section or equivalent, and then click on it to see its specific IP address.
```

Comment:
Line 78: Usage
```suggestion
To do this, go to the IP address of your LAN modem (this may be Googled or requested from your internet provider if you don’t have it) access the app of your local LAN modem, find the iPollo machine in the “connected devices” section or equivalent, and then click on it to see its specific IP address.
```